transcribed/page0680.txt

```diff
@@ -28,7 +28,7 @@
 028     070 018         AOS TORM(I)
 029     070 028         MOVE B,TTYSTS(I)
 030     001 043 TYP7:   TLNE B,TTYIMO
-031     069 007         JRST TYP1
+031     069 007         JRST TYPI
```

Comment:
This one confused me since there is a TYP1 label on line 009 on this page (068), but the reference makes it clear that it is the one on page 069, which is indeed TYPI.

So in the end I agree! :)

Reply:
Yes, this was a tricky one!  The symbol table at the end of the listing really helps here.